ui/apps/platform/src/types/processIndicator.proto.ts

```diff
@@ -16,7 +16,7 @@ export type ProcessIndicator = {
 export type ProcessSignal = {
     id: string;
     containerId: string;
-    time: string; // ISO 8601 date string
+    time: string | null; // ISO 8601 date string
```

Comment:
This field is declared as `string` in the swagger docs and in the proto file, but sometimes is returned as `null`.

TBD - What other fields can be returned as `null` and how do we verify this?